games-action/abuse/abuse-0.8.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="2D side-scrolling platform game"
+DESCRIPTION="Abuse is a dark 2D side-scrolling platform game developed by Crack dot Com in 1995. It features beautiful lighting, realistic animation and nasty alien-like creatures to destroy. It is now maintained by Sam Hocevar in an attempt to prevent it from vanishing from the Internet."
+HOMEPAGE="http://abuse.zoy.org"
+COPYRIGHT="
+	2005-2001 Sam Hocevar
+	2001 Anthony Kruize
+	1995 Crack dot Com
+	1995 Bobby Prince
+	1995 Crack dot Com
+	1996-2008 Justin Cassidy
+	"
```

Comment:
```diff
-COPYRIGHT="
-	2005-2001 Sam Hocevar
+COPYRIGHT="2001-2005 Sam Hocevar
 	2001 Anthony Kruize
 	1995 Crack dot Com
 	1995 Bobby Prince
 	1995 Crack dot Com
+	1996-2008 Justin Cassidy"
-	1996-2008 Justin Cassidy
-	"
```

Reply:
@alaviss huh, i think i fixed all the issues, but github still says "Changes requested", am i missing something?

Reply:
@pgrobelniak when @alaviss is pleased he will remove it :)